test/parallel/test-fs-buffertype-writesync.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const common = require('../common');
+
+// This test insures that writeSync does support input buffers which
```

Comment:
@addalex I amended the comment [here](https://github.com/nodejs/node/pull/12355/commits/38c76eedb85c0f705c37b2fb917576adc6c82761#diff-4bc9a55f999c2e4c2f34c4279e8207c2R4).